ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplExporter.java

```diff
@@ -30,12 +30,12 @@
 public class AmplExporter implements Exporter {
 
     public static final String EXPORT_RATIOTAPCHANGER_VT = "iidm.export.ampl.export-ratio-tap-changer-voltage-target";
-    public static final String SPECIFIC_COMPATIBILITY = "iidm.export.ampl.specific-compatibility";
+    public static final String T2WT_SPLIT_SHUNT_ADMITTANCE = "iidm.export.ampl.t2wt-split-shunt-admittance";
 
     private static final Parameter EXPORT_RATIOTAPCHANGER_VT_PARAMETER = new Parameter(EXPORT_RATIOTAPCHANGER_VT, ParameterType.BOOLEAN, "Export ratio tap changer voltage target", Boolean.FALSE)
             .addAdditionalNames("iidm.export.ampl.exportRatioTapChangerVoltageTarget");
-    private static final Parameter SPECIFIC_COMPATIBILITY_PARAMETER = new Parameter(SPECIFIC_COMPATIBILITY, ParameterType.BOOLEAN, "Export specific compatibility", Boolean.FALSE)
-            .addAdditionalNames("iidm.export.ampl.specificCompatibility");
+    private static final Parameter T2WT_SPLIT_SHUNT_ADMITTANCE_PARAMETER = new Parameter(T2WT_SPLIT_SHUNT_ADMITTANCE, ParameterType.BOOLEAN, "Export t2wt split shunt admittance", Boolean.FALSE)
+            .addAdditionalNames("iidm.export.ampl.t2wtSplitShuntAdmittance");
```

Comment:
Additional names are here only to keep compatibility with deprecated parameters' names. Instead of `iidm.export.ampl.t2wtSplitShuntAdmittance` which had never been used as parameter before, you can put `iidm.export.ampl.specific-compatibility` and `iidm.export.ampl.specificCompatibility`.

Reply:
Done. I Kept  `iidm.export.ampl.t2wtSplitShuntAdmittance` as additional name allowing to write the parameter with dashes or in camelCase (as in `EXPORT_RATIOTAPCHANGER_VT`) . Please confirm if i should delete it.

Reply:
I'd rather delete the camel case one since we wanted to exclusively use snake case in configuration files (except for parameters that can be passed in JSON files).

@mathbagu I think we discussed it (cf issue #219) so correct me if I'm wrong

Reply:
Deleted